ibllib/oneibl/data_handlers.py

```diff
@@ -973,8 +974,9 @@ def setUp(self, task):
                     Path(self.root_path.joinpath(file_uuid)))
             except FileExistsError:
                 pass
-
         task.session_path = SDSC_TMP.joinpath(session_path)
+        assert self.getOutputFiles(session_path=task.session_path).shape[0] == 0, (
+            "On SDSC patcher, input files output files should be distinct from input files to avoid overwriting")

```

Comment:
Minor typo here. Also could you please add a comment about copying input files that are also output file in the future? Otherwise if we encounter this assertion in the future we may not remember the implications of breaking it